docs/specs/toc/service-page.yml

```diff
@@ -789,7 +789,7 @@ outputs:
     {"title": "hello world"}
   a/api/bot/toc.json:
   .errors.log: |
-    {"message_severity":"warning","code":"publish-url-conflict","message":"Two or more files of the same version('v-0.2') publish to the same url '/docs/': 'test/index.yml [generated]<'v-0.2', 'v-0.3'>', 'test1/index.md<'v-0.1', 'v-0.2'>'."}
```

Comment:
Should we keep the original error message as it contains more specific version information?

Reply:
That would require an extra code path to retrieve the monikers. This is a special code path for generated contents (service page) in a corner case, and I feel the message here already conveys the meaning.